front/package.json

```diff
@@ -151,6 +151,9 @@
     "vite-tsconfig-paths": "^5.0.1",
     "vitest": "^2.0.0"
   },
+  "optionalDependencies": {
+    "@rollup/rollup-linux-arm64-gnu": "^4.23.0"
```

Comment:
Could this be split into its own commit? It's unrelated to the bug this PR fixes.

Reply:
Oh, I just noticed @clarani has made the same request above.

Small note: your new commit is missing the "front:" prefix.